lib/mercury/received_message.rb

```diff
@@ -38,14 +38,12 @@ def nack
     private
 
     def is_ackable
-      @work_queue_name != nil
+      !@work_queue_name.nil?
     end
 
     def performing_action(action)
-      is_ackable or raise "This message is not #{action}able"
-      if @action_taken
-        raise "This message was already #{@action_taken}ed"
-      end
+      is_ackable || raise("This message is not #{action}able")
+      raise "This message was already #{@action_taken}ed" if @action_taken
```

Comment:
I often find the nesting is more readable. I like to be able to skip over big strings of text, so I don't like putting a small `if` at the end of a `raise`. When asked to shrink them down to one line, I often just move it out to a method, like `verify_ackable` or `verify_action_taken`.